### ISSUE TITLE
Uncaught Error: Assertion failed: cannot set to grabbable if already set that way

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#622]
Caught while running ?fuzz testing with multiple pointers (`fuzzPointers=2`).

```
Uncaught Error: Assertion failed: cannot set to grabbable if already set that way
Error: Assertion failed: cannot set to grabbable if already set that way
    at window.assertions.assertFunction (http://10.0.0.179:8080/assert/js/assert.js:22:13)
    at GrabDragInteraction.releaseDraggable (http://10.0.0.179:8080/scenery-phet/js/accessibility/GrabDragInteraction.js:443:15)
    at PressListener.release [as _releaseListener] (http://10.0.0.179:8080/scenery-phet/js/accessibility/GrabDragInteraction.js:393:16)
    at PressListener.onRelease (http://10.0.0.179:8080/scenery/js/listeners/PressListener.js:619:10)
    at Action.execute (http://10.0.0.179:8080/axon/js/Action.js:225:18)
    at PressListener.release (http://10.0.0.179:8080/scenery/js/listeners/PressListener.js:418:25)
    at PressListener.interrupt (http://10.0.0.179:8080/scenery/js/listeners/PressListener.js:475:12)
    at PressListener.pointerCancel (http://10.0.0.179:8080/scenery/js/listeners/PressListener.js:754:12)
    at Input.dispatchToListeners (http://10.0.0.179:8080/scenery/js/input/Input.js:1852:25)
    at Input.dispatchEvent (http://10.0.0.179:8080/scenery/js/input/Input.js:1806:10)

```

Seen so far in BASE, friction, coulombs-law.

**Comment by jessegreenberg** [comment]
From https://github.com/phetsims/aqua/issues/106, this should be addressed before continuing there as this was one of only a few issues with fuzzing with multiple pointers.

**Comment by jessegreenberg** [comment]
It doesn't have t happen from `interrupt`, just saw it happen from `up` as well.

**Comment by jessegreenberg** [comment]
Hmm I found one way to get this to happen, at least in BASE.
- Drag balloon with your finger.
- Hit ResetAllButton with second finger while keeping first down.
- Release balloon.

**Comment by jessegreenberg** [comment]
At first, I thougth we needed a way to interrupt the PressListener of GrabDragInteraction.  But then I releasized the above commits would still be broken if client happened to call `GrabDragInteraction.reset` before `GrabDragInteraction.interrupt`.  So https://github.com/phetsims/balloons-and-static-electricity/commit/3cb20e83937b7433daaa43f8fcbc8d882c838862 adds a check to the interrupt to make sure that the node is draggable before reverting to grabbable.

After that, the `interrupt` isn't required in addition to `reset` for most use cases. But I thought it made sense to keep it for completion, so we can interrupt the GrabDragInteractin without fully resetting it.

This fixes the issue for BASE, but ill check on friction and ISLC sims too.

**Comment by jessegreenberg** [comment]
They are indeed fixed with `?fuzz&fuzzPointers=2`. @zepumph could you please review?

**Comment by zepumph** [comment]
Looks good to me, thanks for doing this! Back to you to check on CT later.

**Comment by jessegreenberg** [comment]
Thanks for reviewing! This wasn't on CT but was caught during https://github.com/phetsims/aqua/issues/106. Not seeing this locally anywhere anymore so closing.
